### ISSUE TITLE
Screen reader is not announcing the expand/ collapse state when focus is on "sub menu" button in the command bar.

### ISSUE THREAD

**Issue by AbdelrhmanMagdy** [OfficeDev/office-ui-fabric-react#11553]
### Environment Information
- **Package version(s)**: 7.75.0
- **Browser and OS versions**: Chrome & Windows

### Describe the issue:
Screen reader is not announcing the expand/ collapse state when focus is on "sub menu" button in the command bar.

[Codepen reproduction](https://codepen.io/abdelrhmanmagdy/pen/GRgEoqz?editors=1010)
Just open the code pen and open the NVDA or any screen reader and go to the button then click enter and observe the issue.

#### Actual behavior:
when i clicked and open the sub menu button the screen reader reads the first element without announcing that the sub menu is expanded.

#### Expected behavior:
when i click on the sub menu button the NVDA should announce that the sub menu is expended and when i close the sub menu NVDA should announce that the sub menu is collapsed.
This expected behavior is common behavior and it's actually used in the office fabric portal header.

**Comment by micahgodbolt** [comment]
Does this currently work in Narrator?

**Comment by AbdelrhmanMagdy** [comment]
> Does this currently work in Narrator and Edge?

it's only declare the collapsed state of the dropdown

**Comment by micahgodbolt** [comment]
I don't see any examples in w3 that contain more narrator information than our menu buttons. On focus you get that it is a button, and expanded="false".

When taking expected 'expanding' action (enter, or alt down arrow) the menu opens and narrator tells you that you're on a menu item. Upon closing focus goes back to the button and state is reported as 'expanded=false'. 

If you can point me to a better example I can see what I can do to match that behavior.
